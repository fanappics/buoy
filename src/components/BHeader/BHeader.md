This is what the BHeader looks like:
```js
<b-header class="edit_me">Iâ€™m a header!</b-header>
```
