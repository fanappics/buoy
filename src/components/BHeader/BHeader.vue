<template>
  <header>
    <slot></slot>
  </header>
</template>

<script>
  /**
  * Buoy Header Component
  */
export default {
  name: 'b-header'
}
</script>

<style scoped>
header {
  background-color: rgb(248,248,248);
  display: flex;
  flex: 0 0 3rem;
  flex-direction: column;
  font-weight: bold;
  justify-content: center;
  padding: 0rem 2rem;
}
</style>
<docs>
Sample documentation usage within `.vue` file

```js
<b-header class="edit_me">Iâ€™m a header!</b-header>
```

</docs>
