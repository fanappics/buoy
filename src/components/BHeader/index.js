import BHeader from './BHeader.vue'

/* istanbul ignore next */
BHeader.install = function install (Vue) {
  Vue.component(BHeader.name, BHeader)
}

export default BHeader
