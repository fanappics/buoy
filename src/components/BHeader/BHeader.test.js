import { mount } from 'vue-test-utils'
import BHeader from './BHeader'

describe('BHeader', () => {
  it('renders correctly', () => {
    const slot = `<div />`
    const wrapper = mount(BHeader, {
      slots: { default: slot }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
