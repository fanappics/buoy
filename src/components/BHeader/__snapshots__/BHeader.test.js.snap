// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BHeader renders correctly 1`] = `
<header>
    <div></div>
</header>
`;
